### ISSUE TITLE
[BUG]Keyword "2Theta" necessary when using compute xrd command

### ISSUE THREAD

**Issue by davidfir3** [lammps/lammps#2814]
**Summary**

I was trying to use "compute xrd" command to output XRD pattern of crystals. The result shows that diffraction intensities within the whole scattering angle range are all zero, which is impossible. Parts of my script as below:
```
compute myxrd all xrd 1.541838 C echo
fix 1 all ave/histo/weight 1 1 1 1 179 178 c_myxrd[1] c_myxrd[2] mode vector file crystal.xrd
run 0
```
However, when I append keyword "2Theta 1 179 " to compute xrd command instead, the output result shows correct diffraction intensities. Changed command as below:
`compute myxrd all xrd 1.541838 C 2Theta 1 179 echo`

Hence, it seems that "2Theta" keyword is necessary when using compute xrd command even if the default values, "2Theta 1 179" here, are desired.

**LAMMPS Version and Platform**

LAMMPS (27 May 2021)
Linux Ubuntu 20.04

**Expected Behavior**

Valid intensity values were output without optional keyword "2Theta".

**Actual Behavior**

All intensity values are zero without keyword "2Theta", while valid intensity values are output with "2Theta 1 179" appended to compute xrd.

**Steps to Reproduce**

Run system.lmp in the attached zip file

**Further Information, Files, and Links**

[system.zip](https://github.com/lammps/lammps/files/6724584/system.zip)

**Comment by akohlmey** [comment]
Thanks for the report. This should be addressed with commit 4a21a0e727a8ea8d8cb8ea7d2070e0a0ea5482c8 included in pull request #2813. Until this branch is merged and a new patch released the 2Theta keyword is required as observed.

**Comment by davidfir3** [comment]
@akohlmey Thanks for the reply.
